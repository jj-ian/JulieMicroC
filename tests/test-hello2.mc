int lol() {
    print(42);
    print(71);
    print(1);
 /*   lol();*/
    return 0;
}

int main() {
    print(2);
    return 0;
}
