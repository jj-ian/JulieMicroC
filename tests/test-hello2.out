42
71
1
2
